### PR TITLE
Update 0x02c-Acknowledgements.md

### PR DESCRIPTION
**Pull request by Laancelot**
Typos and inconsistencies fixed - also, the meetup link I removed doesn't work

Thank you for submitting a Pull Request to the OWASP MASTG. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR closes #< insert number here >.

### CODE REVIEW

File: Document/0x02c-Acknowledgements.md [hunk 1]
```diff
@@ -43,8 +43,8 @@ If you'd like to apply please contact the project leaders by sending an email to
 - [Financial donations](https://mas.owasp.org/donate/) are not part of the eligibility criteria but will be listed for completion.
 - Re-shared publications and blog posts linked in MASTG text must be **educational** and focus on mobile security or MASVS/MASTG and **not endorse company products/services**.
 - Advocate Companies may use the logo and links to MASVS/MASTG resources as part of their communication but cannot use them as an endorsement by OWASP as a preferred provider of software and services.
-  - Example of what's ok: list MAS Advocate” status on website home page, in about company slides in sales presentations, on sales collateral.
-  - Example of what's not ok: an MAS Advocate” cannot claim they are OWASP certified.
+  - Example of what's ok: list MAS Advocate status on website home page, in "about" company slides in sales presentations, on sales collateral.
```
Comment (TheDauntless) [inline Document/0x02c-Acknowledgements.md:46]:
```suggestion
  - Example of what's ok: list MAS Advocate status on website home page, in "about company" slides in sales presentations, on sales collateral.
```
Seems more logical.

File: Document/0x02c-Acknowledgements.md [hunk 2]
```diff
@@ -93,8 +93,8 @@ In the past, NowSecure has also contributed to the project, has sponsored it bec
   - [Integrate security into the mobile app software development lifecycle](https://www.scmagazine.com/perspective/devops/integrate-security-into-the-mobile-app-software-development-lifecycle)
   - [OWASP Mobile Security Testing Checklist Aids Compliance](https://www.nowsecure.com/blog/2022/02/23/owasp-mobile-security-testing-checklist-aids-compliance/)
 - **Presentations**:
-  - “Mobile Wanderlust”! Our journey to Version 2.0! (OWASP AppSec EU, Jun 10 2022
-  - [Insiders Guide to Mobile AppSec with Latest OWASP MASVS (OWASP Toronto Chapter, Feb 10 2022)](https://www.meetup.com/owasp-toronto/events/283383269/)
+  - “Mobile Wanderlust”! Our journey to Version 2.0! (OWASP AppSec EU, Jun 10 2022)
```
Comment (TheDauntless) [inline Document/0x02c-Acknowledgements.md:96]:
```suggestion
  - "Mobile Wanderlust"! Our journey to Version 2.0! (OWASP AppSec EU, Jun 10 2022)
```
Suggest using normal double quotes.

**Review by TheDauntless — COMMENTED** [review]
Hi Sir Lancelot!

Thanks for contributing. I made 2 tiny suggestions; are these ok for you?

**Review by cpholguera — APPROVED** [review]
Thank you very much @Laancelot!

**Pull request merged**